kong/plugins/key-auth/schema.lua

```diff
@@ -52,5 +52,9 @@ return {
       type = "boolean",
       default = false,
     },
+    run_on_preflight = {
+      type = "boolean",
+      default = true,
```

Comment:
The issue with such "on by default" booleans is that they need a migration to be inserted.

Reply:
see other comment, a follow up PR.